src/scripts/main.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
-// write code here
+const baseUrl = 'https://mate-academy.github.io/'
+  + 'phone-catalogue-static/api/phones';
```

Comment:
```suggestion
const BASE_URL = 
  'https://mate-academy.github.io/phone-catalogue-static/api/phones';
```
Краще перенести на два рядки, ніж використовувати конкатинацію для оголошення URL